### DISCUSSION TITLE
contribution activity is not working with secondary email registered in my github profile

### DISCUSSION THREAD

**Discussion by lepinekong** [community/community#22886] [New to GitHub]
<p>I have checked that email configured for git repo is same as secondary email registered on github profile, but after pushing to github, activity is not reflecting it.</p>
<p>Is this normal ?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Check this out: <a href="https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/" rel="nofollow noopener">https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/</a></p>
